src/Compilers/CSharp/Portable/Symbols/NativeIntegerTypeSymbol.cs

```diff
@@ -35,22 +40,29 @@ internal NativeIntegerTypeSymbol(NamedTypeSymbol underlying) : base(underlying,
 
         public override SpecialType SpecialType => _underlyingType.SpecialType;
 
-        public override IEnumerable<string> MemberNames => Array.Empty<string>();
+        public override IEnumerable<string> MemberNames => GetMembers().SelectAsArray(m => m.Name);
 
         /// <summary>
-        /// There are no members on <see cref="System.IntPtr"/> that should be exposed directly on nint:
+        /// There only members of <see cref="System.IntPtr"/> that are exposed directly on nint
+        /// are the overridden methods Equals(), GetHashCode(), and ToString(). For the other members:
```

Comment:
>are the overridden methods Equals(), GetHashCode(), and ToString() [](start = 12, length = 66)

Should we instead build the set dynamically by picking up all members that are overriding members from underlying?